kythe/extractors/openjdk11/README.md

```diff
@@ -0,0 +1,21 @@
+# Kythe Extracting openjdk11
+
+This package describes extracting OpenJDK11 from source using Kythe
+
+## Fetch and Configure OpenJDK11
+
+Follow the instructions outline at
+http://hg.openjdk.java.net/jdk/jdk11/raw-file/tip/doc/building.html
+for configuring OpenJDK11 for your platform.
+
+## Extraction
+
+Once you have a correctly configured source tree for your platform,
+run:
+
+```
+$ bazel run //kythe/extractors/openjdk11:extract -- path/to/openjdk11
```

Comment:
Path should be absolute because bazel changes the cwd.

Reply:
Done.